### ISSUE TITLE
Size of tested datasets

### ISSUE THREAD

**Issue by liu431** [uchicago-computation-workshop/marc_berman#5]
Hello Marc,
The paper makes excellent illustration of the datasets' degree of fit with the DFA model using large R<sup>2</sup>. However, I'm concerned about the potential problems of relatively small size of the datasets. For example, MTSA included only 19 adults, SART and TMT included 27, and VWMT included 30. Is this mainly because of the high cost of fMRI scan experiment?

**Comment by bermanm** [comment]
Yes, this is pretty standard for fMRI experiments.  I think we can take comfort that many of the results replicated across studies, but you are right that fMRI studies often suffer from small sample sizes because the experiments run about $600-700 per participant.
